core/templates/services/services.constants.ts

```diff
@@ -69,4 +69,6 @@ export class ServicesConstants {
 
   public static RTE_COMPONENT_SPECS = (
       require('rich_text_components_definitions.ts'));
+
+  public static WHITELISTED_IFRAME_HOSTS = ['mockWindow'];
```

Comment:
assets.constants.ts is where to declare this sort of thing, which should be [] by default. (We'll swap it out during the deployment process.)

Reply:
currently has a fake value for testing purposes.